csm_web/scheduler/models.py

```diff
@@ -15,6 +15,26 @@ def can_enroll_in_course(self, course):
         )
 
 
+def bio_image_upload_path(instance, _filename):
+    # Get the appropriate semester abbreviation
+    today = timezone.now().date()
+    semester = "fa" if today.month > 6 else "sp"
+    return semester + today.strftime("%y") + "/" + str(instance.user.pk)
+
+
+class MentorBioInfo(models.Model):
+    """
+    Includes biographical information about a mentor. Only one should exist per mentor;
+    edits to the bio should edit the existing instance in the database.
+    """
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
This seems to rely on mentors having `Mentor` objects in the database so you can associate their bio with the appropriate course(s). But I can see several scenarios where this assumption doesn't hold.
1. Content Mentors currently have bios on the website, but are not on Scheduler
2. Senior Mentors who are not also teaching a section
3. Members of Exec who don't have `Coordinator` objects in the DB because they don't need Scheduler access (like the socials chair, EVP/IVP/President potentially depending on how hands-on they are)

You can't just create "dummy profiles" to associate CMs/SMs with a course because they are related to a course via their section, and then we'd have to create "dummy sections" but have some way of identifying them so they are filtered out of our normal API endpoints and that sounds really hacky.